pkg/build/base/base.go

```diff
@@ -24,26 +24,20 @@ import (
 	log "github.com/sirupsen/logrus"
 
 	"sigs.k8s.io/kind/pkg/build/base/sources"
-	"sigs.k8s.io/kind/pkg/docker"
 	"sigs.k8s.io/kind/pkg/exec"
 	"sigs.k8s.io/kind/pkg/fs"
 )
 
-// DefaultImageName is the default name of the built base image
-const DefaultImageName = "kindest/base"
-
-// DefaultImageTag is the default tag of the built base image
-const DefaultImageTag = "latest"
+// DefaultImage is the default tag of the built base image
```

Comment:
nit: default "name"

Reply:
er docker is super unclear on this, it's the `-t` (tag) argument when building. /shrug